src/components/Create/components/pages/ReviewDeploy/components/RulesReview/RulesReview.tsx

```diff
@@ -18,7 +18,7 @@ export const RulesReview = () => {
   return (
     <div className="flex flex-col gap-y-10 pt-5 pb-8 md:grid md:grid-cols-4">
       <ReviewDescription
-        title={t`Reconfiguration`}
+        title={t`Edit deadline`}
```

Comment:
I wonder if this should be name `Cycle edit deadline`. Otherwise it might not be immediately clear what the edit deadline is referring to.

Reply:
Good idea. I'll think about this one — possibly for a future PR.